src/viam/sdk/CMakeLists.txt

```diff
@@ -289,9 +289,14 @@ endif()
 
 if (APPLE)
   target_link_libraries(viamsdk PUBLIC "-framework Security")
-else()
+elseif (NOT WIN32)
```

Comment:
I think that some of these should actually be declared as `target_link_libraries` on `viam_rust_utils`, if that is the library that actually brings these dependencies in.

Reply:
Could you, before merging, add a note here explaining that we would have liked to attach several of these dependencies to `viam_rust_utils`, but currently lack an understanding of how to do so?